src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -0,0 +1,139 @@
+    <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+
+    <style amp-custom>
+    form.amp-form-submit-success [submit-success] {
+        width: 100%;
+        color: black;
+    }
+    form.amp-form-submit-success > div.comment-user, 
+    form.amp-form-submit-success .button{
+        display: none;
+    }
+    form.amp-form-submit-error [submit-error] {
+        color: red;
+    }
+    .user {
+      color: #ff4081;
+      font-weight: 700;
+    }
+    .date {
+      color: #a8a3ae;
+    }
+    .comment {
+        background: #f2f2f2;
+        margin: 0 16px 16px 0;
+        width: 100%;
+    }
+    .button-primary {
+      margin: 16px
+    }
+    .user-image {
+      margin-right: 16px;
+      flex-shrink: 0;
+    }
+    .comments-heading {
+      padding-top: 16px;
+    }
+    .comments-container {
+      padding-left: 16px;
+    }
+    .comment-user {
+      display: flex;
+    }
+    </style>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- [amp-access](/components/amp-access) requires the definition of 3 endpoints as documented [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md). This sample allows an user to login and logout: this is realized by specifying two different endpoints in the login property, `sign-in` and `sign-out`, find more [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md#login-page)-->
+    <script id="amp-access" type="application/json">
+    {
+        "authorization": "<%host%>/samples_templates/comment_section/authorization?rid=READER_ID&url=CANONICAL_URL&ref=DOCUMENT_REFERRER&_=RANDOM",
+        "noPingback": "true",
+        "login": {
+          "sign-in": "<%host%>/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL",
+          "sign-out": "<%host%>/samples_templates/comment_section/logout"
+        },
+        "authorizationFallbackResponse": {
+            "error": true,
+            "loggedIn": false
+        }
+    }
+    </script>
+    </head>
+    <body>
+    <!-- #### Add a Comment Section -->
+    <h5 class="comments-heading">Comments</h5>   
```

Comment:
Add `<!-- -->` above this line. Otherwise the headline will be shown as a code sample in mobile view. 

Also: why do you define a new rule `comments-heading`? I'd prefer if you use the default styling for things like headlines etc.